Program.cs

```diff
@@ -1,4 +1,30 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+if (int.TryParse (args[0], out int num)) 
+   Console.WriteLine ($"The {num}th Armstrong number is: " + NthArmstrongNumber (num));
+else {
+   Console.WriteLine ("Please provide a positive integer as input.");
+   return;
+}
+static int NthArmstrongNumber (int num) {

```

Comment:
some logic can be added to handle numbers less than 10. Any number less than 10 will have the same number as answer.